### ISSUE TITLE
the difference between NCP and Linear

### ISSUE THREAD

**Issue by Plus-Ultar** [mlech26l/ncps#39]
Hello,sorry to bother you.I want to know whether all the Linear layers in a neural network can be replaced by the NCP model to optimize the network structure.

**Comment by mlech26l** [comment]
Not really. An NCP is a recurrent neural network and can only be used for processing time-series/sequential data, whereas a linear layer can also handle normal non-time-series data. Thus, a linear layer for processing non-time series data cannot be replaced by an NCP (or does not make much sense to replace it).

**Comment by Plus-Ultar** [comment]
Thank you for your clear explanation. At the same time, I wish you a happy Chinese New Year!
